src/Path.php

```diff
@@ -27,13 +27,22 @@ public function relativize(string $absolutePath): string
     {
         $relativePath = str_replace($this->workingDirectory, '.', $absolutePath);
 
-        return is_dir($absolutePath) ? rtrim($relativePath, '/').'/' : $relativePath;
+        return is_dir($absolutePath) ? rtrim($relativePath, DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR : $relativePath;
     }
 
     public function concatenate(array $parts): string
     {
         return array_reduce($parts, function (string $initial, string $next): string {
-            return rtrim($initial, '/').'/'.ltrim($next, '/');
+            if (strlen($initial) === 0) {
```

Comment:
'' === $initial, no need to call a function here, isn't it?

Reply:
Without it, you will always prepend the path with a directory separator

Reply:
Oh sorry, misread, yeah you could replace it with checking if it is an empty string